src/cider/nrepl/middleware/track_state.clj

```diff
@@ -40,10 +40,14 @@
                               [sym (um/relevant-meta the-meta)]))))))))
 
 (defn- cljs-meta-with-fn
-  "Like (:meta m) but adds {:fn true} if (:fn-var m) is true."
+  "Like (:meta m) but adds {:fn true} if (:fn-var m) is true, (:tag m) is
+  'function, or if m does not have a :tag key."
   [m]
   (cond-> (:meta m)
-    (:fn-var m) (assoc :fn true)))
+    (or (:fn-var m)
+        (= (:tag m) 'function)
+        (= (get m :tag :no-tag) :no-tag))
```

Comment:
Normally this would be `(not (contains? m :tag))`.
If that doesn't work for some reason (if `m` is not a collection) the ideally correct way to do this would be with `gensym` (so there's zero risk of mistaking `{}` with `{:tag :no-tag}`).

``` clojure
(let [no-tag (gensym)]
  (cond-> (meta m)
    (or (:fn-var m)
        (= (:tag m) 'function)
        (= (get m :tag no-tag) no-tag))
    (assoc :fn true)))
```

Reply:
Ah, of course. I forgot `contains?` exists. Thanks!